ui/lib/core/addon/components/key-value-header.js

```diff
@@ -1,87 +1,69 @@
-import { computed } from '@ember/object';
-import Component from '@ember/component';
+import Component from '@glimmer/component';
 import utils from 'vault/lib/key-utils';
-import layout from '../templates/components/key-value-header';
 import { encodePath } from 'vault/utils/path-encoding-helpers';
 
-export default Component.extend({
-  layout,
-  tagName: 'nav',
-  classNames: 'key-value-header breadcrumb',
-  ariaLabel: 'breadcrumbs',
-  attributeBindings: ['ariaLabel:aria-label', 'aria-hidden'],
```

Comment:
From what I understand about attribute bindings this read as a conditional. Basically by default aria-label was always attached as an attribute unless `airaLabel` was false. See glimmer [docs here](https://guides.emberjs.com/release/upgrading/current-edition/glimmer-components/). I couldn't find any cases where aria-label should instead by aria-hidden.

Reply:
Note: we followed the classNameBindings pattern in the old version, so it maybe helpful to look at that example on the docs

Reply:
This is now what I have in the .hbs template:
`<nav class="key-value-header breadcrumb" aria-label="breadcrumbs">`

Reply:
This looks right to me. I didn't see anywhere in the code that we passed in `aria-hidden` as an attribute for this component